app/nexus/internal/env/secret.go

```diff
@@ -17,7 +17,7 @@ import (
 // If the environment variable is not set, contains an invalid integer, or
 // specifies a non-positive value, it returns the default of 10 versions.
 func MaxSecretVersions() int {
-	p := os.Getenv(env.NexusMaxSecretVersions)
+	p := os.Getenv(env.NexusMaxEntryVersions)
```

Comment:
Linter complained about the name being "insecure" (false positive) -- so I renamed "secret" as "entry".